common/src/main/resources/startup.properties

```diff
@@ -348,3 +348,5 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 ### LogMoveService Properties
 *.falcon.logMoveService.threadCount=200
 *.falcon.logMoveService.blockingQueue.length=50
+*.falcon.logMoveService.CorePoolSize=20
+##Note CorePoolSize should always be smaller than threadCount.
```

Comment:
More readable if the comment appears before the property. Else, it will appear as if it applies to the next line.